### PR TITLE
Update interaction to create oob otp sms authenticator by whatsapp otp

### PR DESCRIPTION
**Pull request by carmenlau**
refs #1963 

The PR only updates the interaction with the whatsapp otp verification flow, the otp store is not implemented yet so when you click verify it will always succeed. The PR covers adding 1st and 2nd authenticators during signup, at settings page and portal.

### CODE REVIEW

**Review by louischan-oursky — COMMENTED** [review]
1. The verification step during signup does not support whatsapp
2. In adding secondary authenticator, switch to SMS and then switch back to whatsapp requires the user to type the phone number again. I think phone number is fixed at the very first step, re-typing should not be necessary.

File: pkg/auth/handler/webapp/whatsapp_otp.go [hunk 1]
```diff
@@ -5,22 +5,46 @@ import (
...
 var TemplateWebWhatsappHTML = template.RegisterHTML(
 	"web/whatsapp_otp.html",
 	components...,
 )
 
+var FallbackSMSPasswordSchema = validation.NewSimpleSchema(`
+	{
+		"type": "object",
+		"properties": {
+			"x_e164": { "type": "string" }
```
[the remark this answers is not in the document]
Comment (carmenlau) [reply pkg/auth/handler/webapp/whatsapp_otp.go:24]: The phone is for falling back to sms and it is an hidden input in the form. I think I can try get the phone from the last node of graph, let me remove it.

**Comment by carmenlau** [comment]
> The verification step during signup does not support whatsapp

Added to the list in #1963, will handle it separately. 

> In adding secondary authenticator, switch to SMS and then switch back to whatsapp requires the user to type the phone number again. I think phone number is fixed at the very first step, re-typing should not be necessary.

Let me adjust!

**Comment by carmenlau** [comment]
Update in this PR:
- [ ] Setup secondary authenticator alternatives should not show sms if whatsapp is enabled. If user want to use sms, they should click fallback from the whatsapp otp page
- [ ] Remove unnecessary phone input in the whatsapp otp page
- [ ] When user click back, the buttons on the page should be enabled

**Pull request merged**